xskillscore/core/comparative.py

```diff
@@ -197,9 +205,9 @@ def _categorical_metric(observations, forecasts, dim):
 def halfwidth_ci_test(
     forecasts1: XArray,
     forecasts2: XArray,
-    observations: Optional[XArray] = None,
-    metric: Optional[str] = None,
-    dim: Optional[Union[str, List[str]]] = None,
+    observations: XArray | None = None,
```

Comment:
I have no idea what's better between `Optional[XArray] = None` or `XArray | None = None`

Reply:
`XArray | None = None` is the new recommendation. https://www.python.org/dev/peps/pep-0604/

Reply:
In function arguments, you don't have to add | None -- it's implied if you make None the default

def f(arg: int = None) is equivalent

Reply:
thanks implemented the shorter @blackary